internal/rbd/nodeserver.go

```diff
@@ -241,7 +241,11 @@ func populateRbdVol(
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	if req.GetVolumeContext()["mounter"] == rbdDefaultMounter &&
```

Comment:
I might be misreading, but I think you could just switch to querying `rv.Mounter` instead of `req.GetVolumeContext()["mounter"]` here and get rid of the `else` branch entirely.